### PR TITLE
Update OpenCL-cpp to v2.0.16-61a5c9a-p0

### PR DESCRIPTION
**Pull request by rbsheth**
* I've followed [this guide](https://hunter.readthedocs.io/en/latest/creating-new/update.html)
  step by step carefully. **Yes**

### CODE REVIEW

File: examples/OpenCL-cpp/CMakeLists.txt [hunk 1]
```diff
@@ -7,7 +7,10 @@ include("../common.cmake")
 project(download_opencl_cpp)
 
 hunter_add_package(OpenCL-cpp)
-find_package(OpenCL-cpp CONFIG REQUIRED)
+find_package(OpenCLHeadersCpp CONFIG REQUIRED)
+
+hunter_add_package(OpenCL)
```
Comment (NeroBurner) [inline examples/OpenCL-cpp/CMakeLists.txt:12]: could you explain why we need to manually add OpenCL-cpp and OpenCL? Maybe leave the explanation as a comment in the example and additionally maybe add that to the documentation

Comment (rbsheth) [reply examples/OpenCL-cpp/CMakeLists.txt:12]: OpenCL is just the C headers, these are the CPP headers (which depend on the C headers). As usual with KhronosGroup libraries, the naming and interdependencies are confusing. 😕

File: cmake/projects/OpenCL-cpp/hunter.cmake [hunk 2]
```diff
@@ -8,6 +8,17 @@ include(hunter_download)
 include(hunter_pick_scheme)
 
 # List of versions here...
+hunter_add_version(
```
[the remark this answers is not in the document]
Comment (rbsheth) [reply cmake/projects/OpenCL-cpp/hunter.cmake:11]: Sure, we can make that a new best practice

**Review by NeroBurner — APPROVED** [review]
If it is OK for you it only builds on gcc8 and nowhere else, then merge ;)

**Comment by NeroBurner** [comment]
Released: https://github.com/cpp-pm/hunter/releases/tag/v0.24.13

**Pull request merged**